aii-opennebula/src/main/resources/tests/profiles/vm.pan

```diff
@@ -13,6 +13,11 @@ prefix "/hardware";
         "eth2", dict("boot", "false", "hwaddr", "AA:00:00:80:01:02", "pxe", "true"),
         "eth3", dict("boot", "false", "hwaddr", "AA:00:00:80:01:03", "pxe", "true"),
     ),
+    "ib", dict(
+        "ib0", dict("media", "infiniband", "name", "My fancy Infiniband",
+                    "pci", dict("vendor", 0x15b3, "device", 0x1002, "class", 0x0c06),
+        ),
```

Comment:
I know this just a test profile, but it would be easier to read as:

```pan
"ib0", dict(
    "media", "infiniband",
    "name", "My fancy Infiniband",
    "pci", dict(
        "vendor", 0x15b3,
        "device", 0x1002,
        "class", 0x0c06,
    ),
),
```